DOCKER/docker-entrypoint.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+if [ ! -d "$TMHOME/config" ]; then
+	echo "Running tendermint init to create (default) configuration for docker run."
+	tendermint init &&
```

Comment:
Also not critical, but it's nice to define full path for your binaries.
```suggestion
	/usr/bin/tendermint init &&
```

It's hairsplitting, since I wouldn't do the same for `sed`, but I like to keep my binaries well-defined. :)

Reply:
It feels weird to put this in a path that is in `PATH` but then not make use of `PATH`.